Cargo.toml

```diff
@@ -6,8 +6,8 @@ authors = ["Andy Grover <agrover@redhat.com>", "Mulhern <amulhern@redhat.com>",
 [dependencies]
 dbus = "0.5.2"
 clap = "1"
-nix = "0"
-devicemapper = "0.9"
+nix = "0.8.1"
+devicemapper = { path  = "/root/workspace/devicemapper-rs"}
```

Comment:
If you have a PR for this somewhere, you can specify the repo and branch and get the benefit of the CI.

Reply:
thanks. :-)